src/commands/aksCreateCluster/aksCreateCluster.ts

```diff
@@ -40,6 +40,7 @@ export default async function aksCreateCluster(_context: IActionContext, target:
         portalUrl,
         subscriptionTreeItem.subscription.subscriptionId,
```

Comment:
Rather than dereferencing the `subscription` object three times, I think it might be preferable to pass the object itself (i.e. passing `subscriptionTreeItem.subscription` directly).

Reply:
Done, updated.